docs/build-from-scratch.md

```diff
@@ -144,71 +156,63 @@ Azure:
 
   7. AZURE_RESOURCE_GROUP
 
-#### 2. Create resources in a cluster
+#### 2. Create required Velero resources in the cluster
 
-You may create resources on a cluster using our [example configurations][19].
+You can use the `velero install` command to install velero into your cluster, then remove the deployment from the cluster, leaving you
+with all of the required in-cluster resources.
 
 ##### Example
 
-Here is how to setup using an existing cluster in AWS: At the root of the Velero repo:
+This examples assumes you are using an existing cluster in AWS.
 
-- Edit `examples/aws/05-backupstoragelocation.yaml` to point to your AWS S3 bucket and region. Note: you can run `aws s3api list-buckets` to get the name of all your buckets.
-
-- (Optional) Edit `examples/aws/06-volumesnapshotlocation.yaml` to point to your AWS region.
-
-Then run the commands below.
-
-`00-prereqs.yaml` contains all our CustomResourceDefinitions (CRDs) that allow us to perform CRUD operations on backups, restores, schedules, etc. it also contains the `velero` namespace, the `velero` ServiceAccount, and a cluster role binding to grant the `velero` ServiceAccount the cluster-admin role:
+Using the `velero` binary that you've built, run `velero install`:
 
 ```bash
-kubectl apply -f examples/common/00-prereqs.yaml
+# velero install requires a credentials file to exist, but we will
+# not be using it since we're running the server locally, so just 
+# create an empty file to pass to the install command.
+touch fake-credentials-file
+
+velero install \
+  --provider aws \
+  --bucket $BUCKET \
+  --backup-location-config region=$REGION \
+  --snapshot-location-config region=$REGION \
+  --secret-file ./fake-credentials-file
+
+# since we're running the velero server locally, we don't
+# want the in-cluster deployment to be running.
```

Comment:
I wonder if it's worthwhile explaining why that is. I, for one, don't know.

Reply:
If the deployment is running, it will be contending with the local server in trying to process submitted CRDs, leading to unexpected results.

Reply:
sure - the idea is you're trying to run/test the binary that you built locally and is sitting on your laptop, *not* the one that's packaged in `gcr.io/heptio-images/velero:latest`. I'll come up with some wording to explain that a little more.

Reply:
Ahhhh. I didn't know. See if it were me I'd not delete it to find out why. Yeah let's just tell them lol.